dnf-behave-tests/dnf/module/filtering.feature

```diff
@@ -12,8 +13,9 @@ Scenario: Check that only module packages including src are available
         | berry-source | enabled   | wood       |           |
    When I execute dnf with args "repoquery berry"
    Then the exit code is 0
-    And stdout is
+    And stdout matches line by line
```

Comment:
I am fine with the change, I am just curious why is it done?

Reply:
Ah, for some reason I thought the "<REPOSYNC>" is checked only for "stdout matches line by line" and then forgot to check it and change it back. Thanks.